cmd/scollector/collectors/dsc_windows.go

```diff
@@ -0,0 +1,63 @@
+package collectors
+
+import (
+	"os"
+	"time"
+
+	"bosun.org/metadata"
+	"bosun.org/opentsdb"
+)
+
+func init() {
+	collectors = append(collectors, &IntervalCollector{F: c_dsc_mof, Interval: time.Minute * 5})
+}
+
+const (
+	dscLCM = "dsc.lcm."
+	dscMof = "dsc.mof."
+)
+
+var (
+	dscpath     = os.ExpandEnv(`${SYSTEMROOT}\system32\Configuration\`)
+	mapMofFiles = map[string]string{
+		"MetaConfig.mof":       "Meta_Config",
+		"Current.mof":          "Current_Config",
+		"backup.mof":           "Backup_Config",
+		"pending.mof":          "Pending_Config",
+		"DSCStatusHistory.mof": "DSC_History",
+		"DSCEngineCache.mof":   "DSC_Cache",
+	}
+)
+
+// c_dsc_mof monitors the size and last modified time of each mof file.
+// These out of band metrics can be used to verify the DSC WMI Status metrics.
+func c_dsc_mof() (opentsdb.MultiDataPoint, error) {
+	var md opentsdb.MultiDataPoint
+	c := 0
+	if _, err := os.Stat(dscpath + "MetaConfig.mof"); os.IsNotExist(err) {
+		c = 1
+	}
+	Add(&md, dscLCM+"configured", c, nil, metadata.Gauge, metadata.StatusCode, descDSCLCMConfigured)
+	if c == 1 {
+		return md, nil
+	}
+	for filename, filetype := range mapMofFiles {
+		tags := opentsdb.TagSet{"type": filetype}
+		fi, fierr := os.Stat(dscpath + filename)
+		s := int64(-1)
```

Comment:
You could have a separate metric that is 0/1 for existence, and the others just for size. Also, if the MetaConfig.mof file doesn't exist, then none of these will ever run since we return early in that case. Though, maybe that file isn't as short-lived as the others. Up to you. I'm ok leaving it as is.